cves/kernel/CVE-2021-27363.yml

```diff
@@ -157,10 +158,10 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: 2021-03-08
```

Comment:
This provides no real additional information on how the vulnerability was discovered.

Reply:
completely read that wrong! Thought it was asking when. Will fix that in the next commit👍